REVIEWS.md

```diff
@@ -0,0 +1,25 @@
+# Pull Request Reviews
+
+Whether you're contributing or reviewing pull requests, this document can help you!
+
+- If you're reviewing, it can help you understand what to do
+- If you're submitting a pull request, it can help you understand what folks
+are going to do, how to help them, and how to get your pull request
+merged faster
+
+## Steps
+
+We follow a lightweight list of steps in order to review 
+[pull requests in the Athens project](https://github.com/gomods/athens)
+(if you're not familiar with pull requests, see 
+[here](https://help.github.com/articles/about-pull-requests/)).
+
+Here they are:
+
+- Pull requests should be reviewed by at least one 
+[contributor](https://github.com/orgs/gomods/teams/contributors)
```

Comment:
currently this setting requires at least one review from members with write access, contributors only have read access.

Reply:
Fixed